revproxy/views.py

```diff
@@ -89,6 +123,17 @@ def dispatch(self, request, path):
             self.log.exception(error)
             raise
 
+        return proxy_response
+
+    def __set_location_from_original_request(self, request, proxy_response):
+        """This method change the location header from proxy_response to the
```

Comment:
Actually this method replaces the hostname and protocol in a Location header (if present).